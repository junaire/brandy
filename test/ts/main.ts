function getThree(): bigint {
    return 3n;
}

var v = getThree();
console.log(v);
