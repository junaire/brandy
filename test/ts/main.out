@main {
  v: int = call @getThree;
  v: int = id v;
  v1: int = id v;
  print v1;
  v2: int = const 0;
}
@getThree: int {
  v0: int = const 3;
  ret v0;
}
